tests/workbenches/notebook-controller/test_spawning.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright Skodjob authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+#
```

Comment:
we currently do not have license info in our files; any reason to add it?

Reply:
saw no reason to remove it, more like; the code originally comes from odh-e2e repo

guess the world will not end in fire if I delete the header; deleted it now